ui/src/app/utils/runtime-hooks.tsx

```diff
@@ -440,3 +442,30 @@ export const withRuntimeStore = () => (WrappedComponent) => {
     return <WrappedComponent {...props} runtimeStore={value} />;
   };
 };
+
+/**
+ * Hook that initializes the runtime and disk stores and returns their loading states.
+ * Use this to show loading indicators while runtime and disk data are being fetched.
+ * 
+ * @param workspaceNamespace - The namespace of the workspace
+ * @returns Object containing loading states for runtime and disk
+ */
+export const useRuntimeAndDiskStores = (workspaceNamespace: string) => {
```

Comment:
The current disk and runtime are loaded via `useRuntimeStatus` and `useCustomRuntime`.

The individual parts of these hooks' return values could be passed from RuntimeConfigurationPanel to RuntimeConfigurationPanelContents, but that seemed unnecessarily complex.

This new hook serves as a means of doing the setup all at once in order to check to see if state has properly been loaded.